flow_action_components/SendBetterEmail/force-app/main/default/classes/SendBetterEmail.cls

```diff
@@ -131,6 +131,8 @@ public without sharing class SendBetterEmail {
       String replyEmailAddress = curRequest.replyEmailAddress;
       String senderDisplayName = curRequest.senderDisplayName;
       String subject = curRequest.subject;
+      String messageID = curRequest.messageID;
+        System.debug('Message ID = ' + messageID);
```

Comment:
Prolly want to delete this line? [SendBetterEmail.cls]135 (don't need the debug statement in prod?)

Reply:
Good point